http/websockets.go

```diff
@@ -26,6 +27,22 @@ var (
 	cmdNotAllowed     = []byte("Command not allowed.")
 )
 
+var commandShell []string
+
+func init() {
+	if runtime.GOOS != "windows" {
```

Comment:
Should we really add this condition? Filebrowser can be executed inside MSYS/MinGW, where `bash` is available.

Reply:
Yeah, we could just use if it's available.